2021/46xxx/CVE-2021-46380.json

```diff
@@ -1,31 +1,8 @@
 {
```

Comment:
CNA(s) of user (CERTVDE) do not match owning CNA.

Reply:
See CVE Request 1242802